CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
Should we put the v4.0.0 heading and release date in now? Or wait until we get the official green light and do it at the same time as `npm version`?